addons/Popochiu/PopochiuPlugin.gd

```diff
@@ -65,30 +59,36 @@ func _enter_tree() -> void:
 	
 	_export_plugin = preload('PopochiuExportPlugin.gd').new()
 	add_export_plugin(_export_plugin)
-
-
-	## TODO: Clean up when Popochiu configuration is moved to ProjectSettings (refs #26)
+	
+	# TODO: Clean up when Popochiu configuration is moved to ProjectSettings (refs #26)
 	config.ei = _editor_interface
 	config.initialize_project_settings()
-
-
-	for ip in _inspector_plugins:
-		ip.ei = _editor_interface
-		ip.fs = _editor_file_system
-		ip.config = config
-		add_inspector_plugin(ip)
-
-
+	
+	for path in [
```

Comment:
This was something I did and undid for some reasons I can't remember :( It was a good idea after all :sweat_smile:

Reply:
Yup! Sometimes it's good to let your intuition guide you.